geode-core/src/main/java/org/apache/geode/security/SecurityManager.java

```diff
@@ -74,6 +74,8 @@ default void init(Properties securityProps) {}
    * @param principal The principal that's requesting the permission
    * @param permission The permission requested
    * @return true if authorized, false if not
+   *
+   *         possibly throw AuthenticationExpiredException
```

Comment:
Formatting.

Why not list it as a "checked exception". Then use the `@throws` tag to document it.